hedyorg/hedy
# 🪲 fix discrepancy programs count

Fixes #5199 

### How to test?
- login to some user; `user3`
- see the number of created programs in `/users/user3` and `/landing-page`
- they should match.
- run some other programs and perhaps submit some.
- revist the pages, and the numbers should match.
